lib/process-style.js

```diff
@@ -1,10 +1,10 @@
 const cssExtract = require('extract-from-css')
 
-module.exports = function processStyle (stylePart, filePath, config = {}) {
+module.exports = function processStyle (stylePart, filePath, config = {}, jestConfig = {}) {
```

Comment:
Can you remove `config` (it's just jestConfig['vue-jest']) and use jestConfig instead?

Reply:
Done in https://github.com/vuejs/vue-jest/pull/82/commits/abb1460ff2bca2ab00eadcea74c7b19c3b32776b